test/randomized/test_transpiler_equivalence.py

```diff
@@ -199,7 +199,6 @@ def qasm(self):
             st.none(),
             st.sampled_from(mock_backends)),
         opt_level=st.one_of(
```

Comment:
This could be reduced to `opt_level=st.integers(min_value=0, max_value=3)`, but this is also fine as is.